docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -260,6 +260,34 @@ def downstream(may_not_materialize):
     return may_not_materialize + [5]
 ```
 
+### Asset code versions
+
+Assets may be assigned a `code_version`:
```

Comment:
I would give some motivation for why someone would do this: e.g. "Versions let you help Dagster track what assets are _stale_ and avoid performing redundant computation."